index.bs

```diff
@@ -248,6 +248,37 @@ or <dfn for=stream>bidirectional</dfn>.
  </tbody>
 </table>
 
+[=WebTransport stream=] has the following events:
```

Comment:
"events" is the wrong word here, as it has existing meaning in JS specs. How about "signal" (which you use below)?

```suggestion
[=WebTransport stream=] has the following signals:
```